scripts/run.py

```diff
@@ -88,13 +88,13 @@ def visualize():
     Visualizes the benchmark results
     """
     clean_dir("diagrams")
-    set_working_directory("reporting")
```

Comment:
I haven't tried the reporting you built there, but are you sure that all requirements are also met by this reporting tool? I will test it today in the evening, but I would say we could still merge the PR and if really necessary add the old reporting as a second option.